private/bufpkg/buftransport/buftransport.go

```diff
@@ -39,6 +39,11 @@ func SetDisableAPISubdomain(env map[string]string) {
 	env[disableAPISubdomainEnvKey] = "disable"
 }
 
+// TrimAPISubdomain removes the API subdomain from the given address (if present).
+func TrimAPISubdomain(address string) string {
```

Comment:
We just use this in an error message, do we want to have a helper function that has this behavior? This made me wonder why we want to trim a subdomain, which worried me.

Reply:
Yeah the only reason it's there is because this file has the subdomain.  I could just move this to the errors.go file and export the `apiSubdomain`

Reply:
Ok done.  Let me know if this seems better.